test/test_modules.py

```diff
@@ -299,6 +301,33 @@ def test_lstm_net_nobatch(device, out_features, hidden_size):
     torch.testing.assert_close(tds_vec["hidden1_out"][-1], tds_loop["hidden1_out"][-1])
 
 
+class TestPlanner:
+    @pytest.mark.skipif(not _has_gym, reason="no gym")
+    @pytest.mark.parametrize("device", get_available_devices())
+    @pytest.mark.parametrize("env_name", ["Pendulum-v1"])
+    def test_CEM_model_free_env(self, device, env_name, seed=0):
+        env = GymEnv(env_name, frame_skip=1, device=device)
```

Comment:
can't we use our mocking envs? They're stateless, + gym should fail with this (and it restricts the number of tests where this will run)

Reply:
Yes this was a placeholder that wasn't working anyway. I will use the mocking env instead. Once MBEnv is merged into main will also add one with DummyMBEnv